onnxruntime/core/providers/dml/DmlExecutionProvider/src/External/DirectMLHelpers/DmlSerializedGraphDesc.h

```diff
@@ -0,0 +1,73 @@
+//-----------------------------------------------------------------------------
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//
+//-----------------------------------------------------------------------------
+
+#pragma once
+
+struct ConstantName
+{
+    std::string name;
+};
+
+struct ConstantData
+{
+    byte* data;
+    uint32_t size;
+};
```

Comment:
```
struct ConstantData
{
    byte* data;
    uint32_t size;
};
```

```
struct DmlInputSerializedGraphEdge
{
    UINT GraphInputIndex; 
    UINT ToNodeIndex; 
```

Excluding the public API header DirectML.h which has its own Windows-ish rules, do we always use `lowerCamelCase` or `UpperCamelCase` for struct fields within DML and the DML EP?